doc-source/notifiers.rst

```diff
@@ -321,6 +321,33 @@ with the ``-C`` option when executing your compliance checks::
     }
   }
 
+GH Issue Per _Specific_ Check
+******************************
+You can send a notification only for a specific check or set of checks by adding
+a `checks` element to the notifier::
+
+  {
+    "notify": {
+      "gh_issues": {
+        "accr1": {
+          "repo": ["my-org/accr1-repo"],
+          "project": {"Super cool project": "Triage"},
+          "status": ["fail", "error"],
+          "checks": ["some.test.to.create_an_issue_for"]
+        },
+        "accr2": {
+          "repo": ["my-org/accr2-repo"],
+          "project": {"Some other super cool project": "Backlog"},
+          "status": ["error"],
+          "checks": [
+            "some.test.to.create_an_issue_for2",
+            "some.test.to.create_an_issue_for3"
+          ]
+        }
+      }
+    }
+  }
```

Comment:
Rather than this having its own section, can you just fold this into the above **GH Issue Per Check** section?  It seems like it belongs as part of that.  Just add a sentence like `You can also optionally limit your notifications to a set of checks within an accreditation by providing a list of check paths.` and then just add the "checks" JSON to the example above.  I think it would also be useful for the example check paths that you provide to look like similar to the check paths in the controls.json so people can correlate the two.  so instead of `"some.test.to.create_an_issue_for"` you'd have something like `"chk_pkg.chk_cat_foo.checks.chk_module_foo.FooCheckClass"`.  See: https://complianceascode.github.io/auditree-framework/design-principles.html#check-execution

Reply:
OK, done. Thanks!